### PR TITLE
API 8653 controller its

### PR DESCRIPTION
**Pull request by bschofield-va**
- wip
- .
- .
- error checking
- .
- .
- .
- .
- unit tests for controller
- .
- .
- Removed try/catch/log/throw from controller
- Removed FILE_NUMBER from Org Transformer REQUIRED_FIELDS
- Test IDs now include organization, empty bundle check has been consolidated to new R4TestSupport utility class

### CODE REVIEW

**Review by ilaflamme-va — DISMISSED** [review]

File: vista-fhir-query-tests/src/test/java/gov/va/api/health/vistafhirquery/tests/r4/OrganizationIT.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import org.junit.jupiter.api.Test;
+
+@Slf4j
+public class OrganizationIT {
+  private final TestIds testIds = VistaFhirQueryResourceVerifier.ids();
+  @Delegate private final ResourceVerifier verifier = VistaFhirQueryResourceVerifier.r4();
+
+  @Test
+  void read() {
+    assumeEnvironmentNotIn(Environment.STAGING, Environment.PROD);
```
Comment (jhulbert-va) [inline vista-fhir-query-tests/src/test/java/gov/va/api/health/vistafhirquery/tests/r4/OrganizationIT.java:22]: She's gonna fail hard in s-lab/lab as well.

File: vista-fhir-query-tests/src/test/java/gov/va/api/health/vistafhirquery/tests/RawIT.java [hunk 2]
```diff
@@ -31,7 +31,7 @@ void clientKeyIsMissing(String path) {
   }
 
   @ParameterizedTest
-  @ValueSource(strings = {"/internal/raw/Organization", "/internal/raw/Coverage"})
+  @ValueSource(strings = {"/internal/raw/Coverage"})
```
Comment (jhulbert-va) [inline vista-fhir-query-tests/src/test/java/gov/va/api/health/vistafhirquery/tests/RawIT.java:34]: We should probably have an IT for Raw Organization.

File: vista-fhir-query-tests/src/test/java/gov/va/api/health/vistafhirquery/tests/r4/OrganizationIT.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+@Slf4j
+public class OrganizationIT {
+  private final TestIds testIds = VistaFhirQueryResourceVerifier.ids();
+  @Delegate private final ResourceVerifier verifier = VistaFhirQueryResourceVerifier.r4();
+
+  @Test
+  void read() {
+    assumeEnvironmentNotIn(
+        Environment.STAGING_LAB, Environment.STAGING, Environment.LAB, Environment.PROD);
```
Comment (jhulbert-va) [inline vista-fhir-query-tests/src/test/java/gov/va/api/health/vistafhirquery/tests/r4/OrganizationIT.java:23]: `assumeEnvironmentIn()` would be better.

Comment (ilaflamme-va) [reply vista-fhir-query-tests/src/test/java/gov/va/api/health/vistafhirquery/tests/r4/OrganizationIT.java:23]: I fixx

**Review by jhulbert-va — APPROVED** [review]

**Pull request merged**